src/config/config.ts

```diff
@@ -180,6 +180,14 @@ const config = convict({
         default: "site-launch",
       },
     },
+    stepFunctions: {
+      stepFunctionsArn: {
+        doc: "Amazon Resource Name (ARN) of the Step Functions state machine",
+        env: "STEP_FUNCTIONS_ARN",
+        format: "required-string",
+        default: "",
```

Comment:
sanity check - this can default to an empty string? should we just default to test value or does the step function actually hit an upstream api

Reply:
i think you misunderstood me here, what i meant was whether we should give a different default value if it's not provided, rather than an empty string. in any case, the default value is just meant to be informative, which is why i was curious as to why an empty string over something like `MOCK_STEP_FUNCTION`

Reply:
Opps yes, thanks for the sanity check! makes sense to put a default value - in this case we can provide the default step functions for this! done here: 90b36c58da73f13922b26d381e89ec2b518290ed